Examples/Tests/reduced_diagnostics_python/PICMI_inputs_reduced.py

```diff
@@ -0,0 +1,163 @@
+#!/usr/bin/env python3
```

Comment:
This file should be in the same directory as the other reduced diagnostic tests, `Examples/Tests/reduced_diags`. This separate directory is not needed.